kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -807,26 +196,32 @@ window.ExerciseView = Backbone.View.extend({
         var self = this;
 
         var defaults = {
-            seed: Math.floor(Math.random() * 200),
-            framework: "khan-exercises"
```

Comment:
Weird, I remember fixing that elsewhere. Ah, maybe just on my temporary builds of Perseus.